### ISSUE TITLE
Base Info

### ISSUE THREAD

**Issue by R4Z0R49** [DayZMod/DayZ#573]
Would be nice, not sure how easy of a fix this would be, but would it be possible to jack up the damage required to destroy walls and gates significantly to the point it is almost impossible without say, a satchel or two if at all? I know for a fact, at least on US 434 , that anybody who knows what they are doing is not making walls. All over the map are just countless failed bases, solely due to the fact that they kept getting broken. If it takes 10 minutes of hatcheting to bring down what might be an hours worth of wall, something is wrong. All these base glitching preventions mean nothing if nobody uses them. I would say either fix this now or just remove them from this build until you get this sorted out, because I gaurentee you, the only purpose it is serving right now is lagging the servers.

Comment by Lukethero

**Comment by R4Z0R49** [comment]
If what your saying is true then bases should be decaying pretty fast to a point no one but new players are using them. Unless every player on the server is a noob everyday.

if the decay is working as it should and im not seeing any reports other wise so i have to presume it is. (gates being the exception).

Or the base system is too easy to use and everyone is building for the sake of building increasing the amount of objects on the map.

**Comment by AsFoxy** [comment]
He's talking about players destroying them, not decay. The bases in their current form may be rather easy to destroy, however i think if we make them tougher they'll just end up becoming too overpowered. Perhaps we might add the tougher metal walls along with gates small enough to not permit access by vehicles.

**Comment by R4Z0R49** [comment]
Wooden Bases are not going to be only c4 to break in you should be able to spend a little time to break in with normal melee weapons as long as they are willing to use resources to do that i don't see a reason to change.

When the metal fences come back in again they will be much harder to destroy but will require alot more work to place and keep maintained. (give or take)

**Comment by Lukethero** [comment]
Alright, but I am not the only one saying this though. The whole entire server has given up on it because noobs from the coast just run up to their walls and hatchet for a few mins and then take their stuff.

As a base owner myself I thought it was pretty balanced when people only used satchels, just like preety much every base oriented arma 2 mod out. If I wanted to conduct a raid, I would just run up to NWAF, loot the baracks and then go make my attack, not just find the closest barn, run over to the base and tape down my mouse button  while I watched a youtube video.

This might have been fine before when the walls came back on server reset but certainly not now that they are gone forever. I think I speak for the whe server when I say something needs to be done.

**Comment by R4Z0R49** [comment]
Wooden fence's can be broken into this will not change ever weather its by explosive only or any damage is something we are still looking at. C4 is the quickest way just now however any type of damage will work if someone wishes to spend time to hack down the wall and your team cant mount the defense required to keep this from happening i say fair play its not a simple 10 hits and the fence is down it takes resources from the player/players taking down the wall(this might need a bit more balance). Considering its just as quick if not quicker to glitch into the wall i say fair play for them even trying to take it down.

From what your saying the issue seems more the players are building bases and then leaving them unattended.

Nothing is safe!!!!!! I'm sure everyone forgets this

**Comment by Lukethero** [comment]
No before the patch people built up bases such as myself. Then when the patch hit, everyone just ditched them because they could not keep rebuilding all the walls that were getting hacked. An entire base at devils that must have taken weeks to build, the whole sections of the wall were just torn right through, and the clan just ditched it because it provided no help to them. 

And the point about staging a defence, surely you know people cannot be on 24/7. 

This most DEFINATELY needs balance, like big time, because if people say that bases do absolutely nothing for protection now, something is wrong.

**Comment by R4Z0R49** [comment]
Again dayz is about the world persistence if you can not mount the defense needed to protect what you build do not build it.

No one player can be on 24/7 but as a group you should be able to in some sort of degree. The server does not stop when any one player logouts.

I agree some changes need to be made to balance a little more however this will not change the core thought about the walls the basic wooden walls will continue to be the same. Metal walls will be different and require only c4 to destroy but will take alot longer to build and maintain.

Hacks & exploits are always being worked on.

**Comment by bchawke** [comment]
"No before the patch people built up bases such as myself. Then when the patch hit, everyone just ditched them because they could not keep rebuilding all the walls that were getting hacked."
Just so you know, Luke, it was never really intended that base walls would respawn, that would be a ridiculous mechanic.  It was an issue that was fixed.

Personally, I think the core problem here (besides the balance which will take time and a lot of finessing), is people's attitudes towards bases.  DayZ mod is all about the "nothing is safe" principle, but the moment you say the word "base", people expect an impenetrable (or very hard to penetrate) fortress.  They spend days gathering resources and making a big base to put stashes, vehicles, and tents in, and then rage when they come in the next day and everything has been raided.  This will always be an issue with DayZ mod base building unless the entire playerbase completely shifts how they look at base building entirely (very doubtful).  We (the Knights) realized that the best way to build a base and keep it is to not lock the doors!  We built a decent sized base next to NWAF and had some amazing fights there.  Our MO was: use the base when we're around it for regrouping, cooking food safely while guarding the gate, sorting gear, sniping bandits entering NWAF, and generally defending it while we were in the area.  When we left, we left it unlocked and guess what, it was still standing every time we came back.

As long as people try to make bases to store items and vehicles for safekeeping they will be unhappy with the entire concept in DayZ mod.  Too much work goes into building something that is easily taken down when you're offline.  The only other thing to do is build the "we're taking over this town" type base, which, while it is perfectly fair and not against the rules, really cheapens the game IMO (you end up with bandits just 3pp camping their base walls shooting anything that comes within 1km of the town).

Honestly, as much as I hate to say it, base building has had a very negative impact on the playerbase in US434.  Now, as I stated above, it's more about how people view, use, and react to the basebuilding than it is the mechanics itself, but there's no question that 434 has had much more griefing, whining, bitching, moaning, drama, and catfights since bases were added to the game.  It's also had a negative impact on squad fights.  Certain squads that used to roam the map getting into fights build a base near a hot spot on the map and literally spend every hour they play camping in it.  Against the rules?  No.  Good for the overall experience of playing DayZ mod?  A lot of people question whether or not base building even belongs in the game.

Personally, I like the idea of base building and I really appreciate the care going into keeping it true to DayZ mod's core values (nothing is permanent), but I do question whether the appropriate balance can ever be found with this mechanic in a game like this.

**Comment by thcNox** [comment]
I would have to agree that bases are way too easy to break into. It was okay in the beginning when people didn't realize just how easy it was and thought you needed at least a couple of satchels. Our squad run a base that gets broken into on average a couple times of week. Of course this happens in the AM's when no one is online. We also frequently raid other bases and think nothing of it. All you need is a bit of food and water and 5-10 min of your time. It's simply too easy to be any fun. No sense of accomplishment from breaking in. I think it's very odd thinking (no offense intended) that the whole base balancing thing relies on someone from your team being on at all times which is not simply realistic. Our squad consists of 6-8 people, mostly Europeans and some Americans and yet we still can't be online all the time. I'm opposed to the idea that someone can spawn in on the coast, spend a few min looting a major town and then be fully equipped to take on any base on the server. It should be and endgame endeavor to break into bases and require high tier loot and at least as much effort put in by the builder.

As an admin over on Dayz Europa I run one of the very few (1-2) consistent bases (by consistent I mean a base that's been maintained/repaired for a few months or more) You see a very clear pattern when it comes to base building. A new group comes on the server decides to try and build a base. It remains untouched for awhile if they are lucky but then get raided 1-3 times. Owners realize how flimsy bases are and can no long be bothered with the upkeep and decided to abandon their base. This process usually takes two weeks to a month. 

In reply to bchawke's post: If the only use for bases is the very limited use you describe in your post (Though on our server even your base wouldn't last very long) I would question why the devs has chosen to spend so much time on this whole base construction affair when its uses are so limited and marginalized while the effort required and upkeep is quite high. Perhaps it's best abandon the concept of bases in vanilla and focus on other things. Currently bases are not balanced and serve only to frustrate the player base. Most of the community seem completely at odds with the devs when it comes to the use of bases.

**Comment by Lukethero** [comment]
@thcNox You bring up a valid point, why is it that a Bambi can loot up on the coast and tear down weeks worth of what should be endgame base material in minutes. Make this a cooperation between friends and crew members. I agree with @bchawke , bases should not be perma-structures,  but let's be honest, adding 15 hatchet swings will not fix anything.

I see three ways of fixing this.
1. Buff the wooden walls damage drastically but increase the materials needed to build them. Sure we can talk about how great base building will be when they add metal walls in a couple months, but that is not now, so lets do something about it until then. This fixes the issue as well as increasing that endgame gap.
2. Get rid of wall damage all together, but add a tear down feature like gates and take a couple sledges or crowbars. This fix probably could not make it in by the next patch but it does fix the issue and adds that clan cooperation to breaking into the bases. Personally I do not like this one much, but it is an idea.
3. Get rid of bases all together. BC has a point, it has caused a lot of strife, and I think I might have liked the mod better before it was filled with worthless scattered chunks of wood all over the map ever since people realized how easy it was to destroy.

https://youtu.be/nKIt2C79Y_ww

Now I highly suggest you watch this video. For me this was the golden age of base building. It was only with the combined planning and use of explosives, that they were able to break into the walls and take the gear, and obviosly resulted in fun fort both sides. Isn't that what the game is about, fun? This is how it should be,  an operation of endgame groups to band together and raid for the possibility of big loot and taking down a base to actually be an accomplishment (except for the fact they respawned back then, but that has since been adressed).

**Comment by R4Z0R49** [comment]
Personally, I like the idea of base building and I really appreciate the care going into keeping it true to DayZ mod's core values (nothing is permanent), but I do question whether the appropriate balance can ever be found with this mechanic in a game like this.
- No idea until we start getting some info about it rather then the normal moaning about everything we do.

Currently bases are not balanced and serve only to frustrate the player base. Most of the community seem completely at odds with the devs when it comes to the use of bases.
- Removes the shackles forcing you to use base items? If you don't like the base system just don't use them but i would kind of like some info from the players to see if what we wont can be reached. 

The only way we can try new stuff is by adding them and seeing what kind of impact they have on the game but i'm afraid everything about dayz has been lost in the current community. From the sounds of it we may as remove everything give everyone unlimited ammo and have done with it.

**Comment by R4Z0R49** [comment]
1. Buff the wooden walls damage drastically but increase the materials needed to build them. Sure we can talk about how great base building will be when they add metal walls in a couple months, but that is not now, so lets do something about it until then. This fixes the issue as well as increasing that endgame gap.
- Sure we can buff the amount np what will it help add one more player knocking down the wall will counteract the extra you need to build.
1. Get rid of wall damage all together, but add a tear down feature like gates and take a couple sledges or crowbars. This fix probably could not make it in by the next patch but it does fix the issue and adds that clan cooperation to breaking into the bases. Personally I do not like this one much, but it is an idea.
- This was once the plan but it feels cheap and crap
1. Get rid of bases all together. BC has a point, it has caused a lot of strife, and I think I might have liked the mod better before it was filled with worthless scattered chunks of wood all over the map ever since people realized how easy it was to destroy.
- Getting rid is easy to do save me a ton of work.

Take everything back out and be done with any kind of base system id be happy with that.

**Comment by R4Z0R49** [comment]
Resolved

**Comment by Lukethero** [comment]
Of coarse you want to add it back in once a happy medium\ new system is in place. Like I said, I thought base building was quite fun before everyone found out about hatchets. Maybe add an option for server owners to activate them or disable them? 
I might even be wrong here, maybe some people still want them ingame just for the sake/ fun of making them.

**Comment by bchawke** [comment]
I think the problem that @R4Z0R49 is pointing out is that there's a severe lack of people testing and providing useful feedback.  Base building is a SUPER finicky thing in this game (a game in which one of the main principles is that nothing is permanent).  Without people contributing to development by being active here and testing, the official releases become the test bed for content as big and integral as base building.  Unfortunately, and by _NO_ fault of @R4Z0R49 or any of the other devs, the iterations of base building that have been introduced have had negative impact on the community due to the imbalance.  People expect features to be fully working and balanced at each release when that's just not the case.  I don't see how they'll find that happy medium or come up with a new system if nobody is willing to test it, so it sounds like he's removing it long term because the player base was the only real test group.

**Comment by Lukethero** [comment]
@R4Z0R49 Maybe just.boost the base damage a little for the next patch, see how it goes? Nothing too drastic but just something to let us test.

**Comment by bchawke** [comment]
Not sure if any of this would work, but here's my idea:

Keep level 1-5 walls as they are now.  Let people hatchet them if they take the time and energy to do it.  Make level 6 walls (they're the top level, right?) SUPER strong - you'd literally have to hatchet for _HOURS_ with a group of guys to get through it, which means that server restarts would keep you from doing this at all.  Finally, up the damage rate of satchel charges an insane amount to ENSURE that one satchel charge will completely level whatever wall it's placed next to.  Would there be any negative consequence to upping the satchel charge damage rate to an insane level?  In other words, is there anything in the game that we _don't_ want a satchel charge to completely destroy in one hit?  I'd imagine it would be pretty realistic for them to level any wall they are next to and render any vehicle completely inoperable that's within the blast radius.

This would make it possible to still break into bases that haven't been leveled up with melee weapons, but if the builders take the time and resources to fully level up their fortress it will take explosives to break in.  Once that balancing is in effect, it's up to the players to be smart enough to NOT padlock their base until they have it fully fortified, and to keep enough supplies on hand hidden somewhere to rebuild walls that have been taken down in their absence.

@R4Z0R49, sorry if these ideas are coming in too late considering your decision today, but I've been tossing it around in my head today and this is all I can come up with.  I'd be more than willing to dedicate more time to playing on a test server if you'd like to remove base building from the game do test scenarios until you deem the mechanic fit to be reintroduced.  That being said, I'm not against removing base building entirely as it seems to be a very difficult task to properly balance it in a game such as this.

**Comment by Lukethero** [comment]
@bchawke That is actually a really good idea, I would take that over removing them.

**Comment by JackMcS** [comment]
I ll write in here to submit my sorrow about deleting the basebuildingsystem!
I know there is/was a loud discussion about it. But haveing smthg like that in the game is an important thing to differ from lets say SA!
For me it is a natural step for players starting to make themselfes some fences to have a little safe place from Bandits or Zombies!
Just like in TheWalkingDead!
Way before there was building fences in vanilla i was hoping that one time there will be smthg like that in the game.
Imagine there would really be a larg group managing to build a wall around a little village somewhere in chernarus and defending it. WOuldnt that be epic?!
I always planned on doin that, but unfortunately havent ever found enough friends to realise that.

Taking the opportunity out once for all makes the game a bit more everage for me at least.
Maybe more ideas for "balancing" to come up soon!
Greez
